databricks/koalas/tests/test_groupby.py

```diff
@@ -121,6 +121,13 @@ def test_nunique(self):
         self.assert_eq(kdf.groupby("a").agg({"b": "nunique"}),
                        pdf.groupby("a").agg({"b": "nunique"}))
 
+    def test_size(self):
+        pdf = pd.DataFrame({'A': [1, 2, 2, 3, 3, 3],
+                            'B': [1, 1, 2, 3, 3, 3]})
+        kdf = koalas.DataFrame(pdf)
+
+        self.assert_eq(kdf.groupby("A").size(), pdf.groupby("A").size())
```

Comment:
pandas returned a series:

```
A
1    1
2    2
3    3
dtype: int64
```

whereas it looks the current API returned a DataFrame.